python/nano/notebooks/pytorch/cifar10/nano-inference-example.ipynb

```diff
@@ -0,0 +1,642 @@
+    "            self.log(f\"{stage}_acc\", acc, prog_bar=True)\n",
+    "\n",
+    "    def validation_step(self, batch, batch_idx):\n",
+    "        x, y = batch\n",
+    "        logits = self(x)\n",
+    "        loss = F.nll_loss(logits, y)\n",
+    "        preds = torch.argmax(logits, dim=1)\n",
+    "        acc = accuracy(preds, y)\n",
+    "        self.log(\"val_loss\", loss, prog_bar=True)\n",
+    "        self.log(\"val_acc\", acc, prog_bar=True)\n",
+    "\n",
+    "    def test_step(self, batch, batch_idx):\n",
+    "        x, y = batch\n",
+    "        logits = self(x)\n",
+    "        loss = F.nll_loss(logits, y)\n",
+    "        preds = torch.argmax(logits, dim=1)\n",
+    "        acc = accuracy(preds, y)\n",
+    "        self.log_dict({'test_loss': loss, 'test_acc': acc})\n",
+    "\n",
+    "    def configure_optimizers(self):\n",
+    "        optimizer = torch.optim.SGD(\n",
+    "            self.parameters(),\n",
+    "            lr=self.hparams.learning_rate,\n",
+    "            momentum=0.9,\n",
+    "            weight_decay=5e-4,\n",
+    "        )\n",
+    "        steps_per_epoch = 45000\n",
+    "        scheduler_dict = {\n",
+    "            \"scheduler\": OneCycleLR(\n",
+    "                optimizer,\n",
+    "                0.1,\n",
+    "                epochs=self.trainer.max_epochs,\n",
+    "                steps_per_epoch=steps_per_epoch,\n",
+    "            ),\n",
+    "            \"interval\": \"step\",\n",
+    "        }\n",
+    "        return {\"optimizer\": optimizer, \"lr_scheduler\": scheduler_dict}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "7e9da53d",
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Global seed set to 7\n"
+     ]
+    }
+   ],
+   "source": [
+    "seed_everything(7)\n",
+    "PATH_DATASETS = os.environ.get(\"PATH_DATASETS\", \".\")\n",
+    "BATCH_SIZE = 64\n",
+    "NUM_WORKERS = 0\n",
+    "data_module = prepare_data(PATH_DATASETS, BATCH_SIZE, NUM_WORKERS)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "b7d4b74a",
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "GPU available: False, used: False\n",
+      "TPU available: False, using: 0 TPU cores\n",
+      "IPU available: False, using: 0 IPUs\n"
+     ]
+    }
+   ],
+   "source": [
+    "trainer = Trainer(progress_bar_refresh_rate=10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "985d681d",
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "pl_model = LitResnet.load_from_checkpoint('checkpoints/renet18_single_none.ckpt')\n",
```

Comment:
Here I think you may need to add some description saying that we are loading a checkpoint produced in the training example.